app/decorators/decidim/user_decorator.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_dependency 'decidim/user'
+Decidim::User.class_eval do
```

Comment:
Can you please what is this decorator overriding?
I mean, explain what it is doing different from the original class.
If all are helper methods you can explain simply that.